clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2013, Red Hat, Inc., and individual contributors
```

Comment:
This class is new enough to deserve a this year here.

Reply:
This is a rename of ClientMappingExternalizer, which was written in 2013.